### DISCUSSION TITLE
Unable to check for new versions

### DISCUSSION THREAD

**Discussion by joehanna** [pawelsalawa/sqlitestudio#3689] [Q&A]
### Details
SQLiteStudio reports an error when checking for new versions.

Per the Status window:-
```
[20:22:06] Unable to check for updates (Cannot retrieve remote tree No package sources set for this application..)
```

### Steps to reproduce
From the Help menu, choose **Check for Updates**.  Alternatively if "Automatically check for updates at startup" is checked in Preferences, simply start up the app.  The outcome is the same in both cases.

### Operating system
Mac OS Catalina 10.15.3

### SQLiteStudio version
Version 3.2.1

**Comment by pawelsalawa** [comment]
The homepage is in process of migration to new hosting, thus the issue. It will be resolved soon.

**Comment by Anh-Nguyen-08231997** [comment]
> The homepage is in process of migration to new hosting, thus the issue. It will be resolved soon.

Hello Paweł Salawa, I have the same issue. Has this issue been fixed yet? Thank you.

**Comment by pawelsalawa** [comment]
I have restored the repo for the time being, but generally updates repository will be completely removed (after some time) according to #3626. Version 3.2.2 will not rely on this repository anymore.
